detect_secrets/core/usage.py

```diff
@@ -309,6 +309,7 @@ def consolidate_args(args):
 
         active_plugins = {}
         disabled_plugins = {}
+        param_from_default = {}
```

Comment:
Maybe `is_default_parameter`, so it's clearer that it contains boolean values?

Reply:
That's a good idea. I would make a suggestion to rename it to `is_using_default_value`